config/initializers/algoliasearch.rb

```diff
@@ -10,12 +10,14 @@ def initialize
   end
 end
 
-if !Rails.env.production? && ENV.fetch('ALGOLIA_APP_ID') == ALGOLIA_PRODUCTION_APP_ID
+if !Rails.env.production? && ENV['ALGOLIA_APP_ID'] == ALGOLIA_PRODUCTION_APP_ID
```

Comment:
Could we use this instead?
```ruby
ENV.fetch(‘VALUE’, ‘’)
```

Reply:
It returns nil if empty so it's useful on line 17 to do `if ENV['ALGOLIA_APP_ID']`
If you prefer I don't mind, I'll change it to `if ENV['ALGOLIA_APP_ID'] == ''`